from ..exceptions import *

class VariableException(FuncyException):
    pass
